### PR TITLE
GeoJSON plugin docs

### PR DESCRIPTION
**Pull request by langsmith**
pr adds docs for the GeoJSON plugin

will soon add more to the `Using the GeoJSON plugin` section

### CODE REVIEW

**Comment by langsmith** [comment]
the plugins page doesn't appear when I test locally 😕  This is my first time adding a plugin with batfish, so I'm sure I'm missing additions in other files. 

cc @cammace @davidtheclark 

screen shot 2017-09-08 at 11 13 15 am

**Comment by davidtheclark** [comment]
@langsmith: Can you try adding it to this list: https://github.com/mapbox/android-docs/blob/0ef1fe685fe85cb31c67a48a2b9cf815a97da35a/batfish.config.js#L23?

**Comment by langsmith** [comment]
It's working @davidtheclark . Thank you! 
screen shot 2017-09-08 at 11 38 25 am

File: src/pages/plugins/overview/geojson.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+A `OnMarkerEventListener` interface is included in the plugin. Implementing the interface and overriding the `onMarkerClickListener()` method in your activity/fragment enables you to easily do what you would like with the selected marker and the GeoJSON feature (a JSONObject) associated with the marker.
+
+```java
+ @Override
+  public void onMarkerClickListener(Marker marker, JSONObject properties) {
+  // Run whatever code that you'd like
+  }
+```
+
+Once the `geoJsonPlugin` object has been built, you have three methods to choose from for retreiving the GeoJSON file
```
Comment (cammace) [inline src/pages/plugins/overview/geojson.md]: retreiving -> retrieving

File: src/pages/plugins/overview/geojson.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+| Method | Description |
+| --- | --- |
+| `setUrl` | Retrieves the GeoJSON file hosted at a particular URL |
+| `setAssetsName` | Retrieves the GeoJSON file which is located in your app's `assets` folder |
+| `setFilePath` | Allows the user to navigate the device's file structure in a Android dialog in order to find the GeoJSON file |
```
Comment (cammace) [inline src/pages/plugins/overview/geojson.md]: `in order to` -> `to`

File: src/pages/plugins/overview/geojson.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+
+## Load listeners
+
+`OnLoadingGeoJsonListener` is the plugin's other interface that you should know about. Its three methods give you fine-grain control over UI and data interaction.
```
Comment (cammace) [inline src/pages/plugins/overview/geojson.md]: I think these two sentences can be combined? Either using `;` or rewriting

File: src/pages/plugins/overview/geojson.md (hunk 2 above)
Comment (langsmith) [reply src/pages/plugins/overview/geojson.md]: fixed

**Comment by langsmith** [comment]
adjustments have been made @cammace

**Pull request merged**